frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -188,6 +188,17 @@ function testMode(actionType, editMode) {
     expect(wrapper.find("#reply-all-radio").props().checked).toEqual(isReplyAllChecked);
     expect(wrapper.find("#forward-radio").props().checked).toEqual(isForwardChecked);
   } else if (actionType === ACTION_TYPE.task) {
-    //TODO jcherry populate this when task editing has been added
+    //set default values when in "create" mode
+    let valTask = "";
```

Comment:
feel like valTask and valReasonsForAction should have something other than just "", since this also happens to be the default value.

Reply:
never mind, my bad